asreview/models/classifiers/lstm_pool.py

```diff
@@ -90,6 +90,7 @@ class LSTMPoolClassifier(BaseTrainClassifier):
     """
 
     name = "lstm-pool"
+    label = "LSTM-pool"
```

Comment:
LSTM with max-pooling layer?

Reply:
Meaning...?

Reply:
Then what about the label for `lstm-base`?